src/stock-operations/add-stock-operation/stock-items-addition-row.component.tsx

```diff
@@ -203,6 +255,10 @@ const StockItemsAdditionRow: React.FC<StockItemsAdditionRowProps> = ({
                           item?.expiration
                         );
                         setStockItemExpiy(item?.expiration);
```

Comment:
I know this is not related to your changes but i guess it should read setStockExpiry. The last r is omitted.

Reply:
Thanks, I will fix that